packages/neuron-ui/src/components/NetworkSetting/index.tsx

```diff
@@ -101,7 +101,11 @@ const NetworkSetting = ({ chain = chainState, settings: { networks = [] } }: Sta
           value: network.id,
           label: (
             <div className={styles.networkLabel}>
-              <p>{`${network.name} (${network.remote})`}</p>
+              <p>{`${network.name} (${
+                network.remote.length > 40
```

Comment:
Why not use `text-overflow ellipsis`?

Reply:
Intercepting the head and tail in this way, the port number information at the end is clearly visible.
<img width="837" alt="image" src="https://github.com/nervosnetwork/neuron/assets/11756869/c08a3164-6af4-40c8-9dff-456fdf9711bb">

Reply:
40 is an experience number?